pkg/apis/rollouts/v1alpha1/types.go

```diff
@@ -338,9 +341,14 @@ type RolloutAnalysisBackground struct {
 
 // RolloutAnalysis defines a template that is used to create a analysisRun
 type RolloutAnalysis struct {
+	// Whether to look for the templateName at cluster scope or namespace scope
+	// Deprecated and will be removed in v0.9
+	// +optional
+	ClusterScope bool `json:"clusterScope,omitempty"`
 	// TemplateName reference of the AnalysisTemplate name used by the Rollout to create the run
 	// Deprecated and will be removed in v0.9
-	TemplateName string `json:"templateName,omitempty"`
+	// +optional
+	TemplateName string `json:"templateName"`
```

Comment:
This TemplateName needs the omitempty flag because the rollout is letting the user pick between using the TemplateName and Templates (with the goal being eliminating TemplateName in the future)